src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -1953,10 +1955,6 @@ private void PopulateCustomAttributeTableRows(ImmutableArray<IGenericParameter>
             this.AddCustomAttributesToTable(GetEventDefs(), def => GetEventDefinitionHandle(def));
 
             // TODO: standalone signature entries 11
-            if (this.IsFullMetadata)
-            {
-                this.AddCustomAttributesToTable(module.ModuleReferences, TableIndex.ModuleRef);
-            }
```

Comment:
I didn't see a new place this call was inserted.  Can you comment on why this delete is okay?